### PR TITLE
Tweak names using follower/following terms

### PR DESCRIPTION
**Pull request by rohanjr**
Follow up to #74 fixing one mistake and then tweaking some CSS selector names.

### CODE REVIEW

File: ui/src/components/UserList.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const UserList: React.FC<Props> = ({users, onFollow}) => {
               <Icon
                 name='add user'
                 link
-                className='test-select-follow-icon'
+                className='test-select-add-user-icon'
```
Comment (hurryabit) [inline ui/src/components/UserList.tsx]: Isn't `add-user` to generic? What about
```suggestion
                className='test-select-follow-user-icon'
```

File: ui/src/components/UserList.tsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const UserList: React.FC<Props> = ({users, onFollow}) => {
                   <Icon
                     name='add user'
                     link
-                    className='test-select-add-friend-of-user-icon'
+                    className='test-select-add-user-following-icon'
```
Comment (hurryabit) [inline ui/src/components/UserList.tsx]: Is that the "follow back" button? Maybe we should put that in the label then.

File: ui/src/components/MainView.tsx [hunk 3]
```diff
@@ -12,8 +12,8 @@ const MainView: React.FC = () => {
   const myUser = myUserResult.contract?.payload;
   const allUsers = useStreamQuery(User).contracts;
 
-  // Sorted list of users that the current user is following
-  const following = useMemo(() =>
+  // Sorted list of users that are following the current user
```
Comment (hurryabit) [inline ui/src/components/MainView.tsx]:
```suggestion
  // Sorted list of the current user's followers
```

File: ui/src/components/UserList.tsx (hunk 2 above)
Comment (rohanjr) [reply ui/src/components/UserList.tsx]: No these are the icons next to users that your followers are following.. kind of an indirect link.

File: ui/src/components/UserList.tsx (hunk 1 above)
Comment (rohanjr) [reply ui/src/components/UserList.tsx]: I just named this based on the face that the icon is literally called an `'add user'` icon by Semantic UI. Also to have consistency with the icons next to the users' following and avoid `test-select-follow-user-following` - that is accurate on the other hand..

Comment (nemanja-da) [reply ui/src/components/UserList.tsx]: @rohanjr This action leads to a user following another user, right? Then @hurryabit 's suggestion is OK

Comment (rohanjr) [reply ui/src/components/UserList.tsx]: Yeah it's fine. I could change to `'test-select-follow-user-icon'` next to your followers and `'test-select-follow-user-following-icon'` next to their following if you both prefer?

Comment (rohanjr) [reply ui/src/components/UserList.tsx]: Remember these are just for testing so I don't think it's a big deal, but I like nice names too

**Pull request merged**